pkg/pipeline/conntrack/conn.go

```diff
@@ -68,7 +68,7 @@ func (c *connType) toGenericMap() config.GenericMap {
 }
 
 func (c *connType) getHash() totalHashType {
-	return copyTotalHash(c.hash)
+	return c.hash
```

Comment:
Since this is a private type and attribute, maybe the `getHash()` method is not needed anymore and you can invoke directly `c.hash`